internal/cli/cli.go

```diff
@@ -60,11 +60,15 @@ type deleteOpts struct {
 	confirm        bool
 	successMessage string
 	failMessage    string
+	authDB         string
 }
 
 // DeleterFromProject a function to delete from the store.
 type DeleterFromProject func(projectID string, entry string) error
```

Comment:
I am not sure about why we are doing all of this before calling the delete function in the store